test/extended/operators/olm.go

```diff
@@ -37,6 +38,52 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			e2e.Logf("Successfully list %s", olmResources[i])
 		})
 	}
+
+	// OCP-24074 Check OLM resources API Version
+	// Author: jiazha@redhat.com
+	versionMap := map[string]string{
+		"operatorgroups":         "v1",
+		"packagemanifests":       "v1",
+		"catalogsources":         "v1alpha1",
+		"subscriptions":          "v1alpha1",
+		"installplans":           "v1alpha1",
+		"clusterserviceversions": "v1alpha1",
+	}
+
+	for k, v := range versionMap {
+		g.It("check API version: "+k, func() {
+			output, err := oc.AsAdmin().Run("get").Args(k, "-n", "openshift-operator-lifecycle-manager", "-o=jsonpath={.items[0].apiVersion}").Output()
```

Comment:
Even though the earlier test does this, I don't think we can continue to expect an instance of each kind to exist in the `openshift-operator-lifecycle-manager` namespace.